webclient/.bowerrc

```diff
@@ -0,0 +1,4 @@
+{
+  "directory": "bower_components",
```

Comment:
And I think it has to stay here instead of being in remoteappmanager/static/components because it will be packaged and minified in the final javascript bundle by "ember build", I may be wrong